product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/monitoring_using_pemx/index.mdx

```diff
@@ -0,0 +1,93 @@
+
+## Historical charts
+
+It displays the historical data of last 15 minutes by default. To view the historical data of a particular time range, customize the time range using a time range picker. These charts display key metrics in single line and bar or multi line and bar form. It provides a concise snapshot of the information such as:
+  - **Memory** (Line chart) &mdash; Displays the historical trend of memory usage percentage over a time period.
+  - **CPU** (Bar chart) &mdash; Displays the historical trend of CPU usage percentage over a time period.
+  - ** Network activity** (Bar chart) &mdash; Displays the historical data transfer to and from the network card per second, over a time period.
+  - **Disk IOPS** (Multi-line chart) &mdash; Displays the historical trends in the number of reds, writes, and total operations on the disk per second, over a time period.
+  - **Transaction per second** (Multi-line chart) &mdash; Displays the historical trends in the number of transactions per second, over a time period.
+
+
+## Features for both types of charts
+
+All these charts has some tools and features that helps getting more information on the metrics or about the chart. The [time range picker](#time-range-picker) helps in viewing the data on these charts for a specific time range interval. The [information tootip](#information-tooltip) helps to view the information for that particular chart. The [charts error state](#charts-error-state) helps to find out the error and provides the option to edit the configurations and fix the error.
+
+### Time range picker
+
+On `Monitoring and Logging` tab, to view the data of a particular time range configure the time range using:
+- the `Last X` combo box
+- the date-time picker
+
+The `Last X` combo box provides several time range options in the drop-down list. Each option in the list is enabled only after that much time duration has elapsed since the PostgreSQL cluster is created. The default time range is of 15 minutes and the maximum is `Last 30 days`.
+
+!!! Note
+The data on the `Monitoring and Logging` tab is refreshed automatically every 30 seconds when the time range is selected from the `Last X` combo box.
+!!!
+
+### Information tooltip
+
+Each chart has a information tooltip that provides a detail description of the chart. Hover the mouse over the tooltip icon on the right side of the chart name to view the information.
+
+### Charts error state
+
+A red warning icon is displayed next to the tooltip if there is any error for that chart. If any of the metric exceeds its threshold it results into an error warning. Clicking on this red icon displays a window with description of the error and an `Edit Cluster` button. The `Edit Cluster` button takes to the edit cluster page. Do the configuration changes based on the specific metric that brings the cluster in a healthy state. The table shows list of different errors and the corresponding solutions.
+
+| Error  | Solution |
```

Comment:
I'm wondering if the user should be aware of the thresholds configured in the system. WDYT?

Reply:
Just in case, it's 80% for the three metrics below.

Reply:
Let me know the details if this need to be added to the docs